frontend_app_kanban/src/css/index.styl

```diff
@@ -3,6 +3,7 @@
 cardWidth = 280px
 
 .kanban
+  z-index 11
```

Comment:
why 11? Add some intel please, traces should be kept regarding z-index or it'll become a mess
explain "is above xxx" and "bellow yyy" and don't forget to add it in https://algoo.tracim.fr/ui/workspaces/49/contents/html-document/19402

Reply:
Done